CRM/Core/OptionGroup.php

```diff
@@ -281,7 +281,7 @@ public static function &valuesByID($id, $flip = FALSE, $grouping = FALSE, $local
    *
    * @param bool $flip
    */
-  public static function lookupValues(&$params, &$names, $flip = FALSE) {
+  public static function lookupValues(&$params, $names, $flip = FALSE) {
```

Comment:
In comments just above it still refers to this as a 'reference array' of names. But looks like a good improvement since the names never get modified.